BetterJoyForCemu/Joycon.cs

```diff
@@ -655,6 +688,8 @@ private struct Rumble {
                 }
             }
 
+            DetectShake();
+
```

Comment:
Is this new line alright? 
I figured it wasn't really part of the physical buttons section.
@Davidobot